packages/aws-cdk-lib/aws-events-targets/lib/ecs-task.ts

```diff
@@ -7,6 +7,20 @@ import { Construct } from 'constructs';
 import { ContainerOverride } from './ecs-task-properties';
 import { addToDeadLetterQueueResourcePolicy, bindBaseTargetConfig, singletonEventRole, TargetBaseProps } from './util';
 
+/**
+  * Tag
```

Comment:
Can you update this description to give details on what this is? Also the
docstring for `key` and `value`

Reply:
Updated the docstrings here